src/components/TodoList/TodoList.tsx

```diff
@@ -2,223 +1,107 @@
 import React from 'react';
+import { useAppSelector } from '../../app/hooks';
+import classNames from 'classnames';
+import { useDispatch } from 'react-redux';
+import { setCurrentTodo } from '../../features/currentTodo';
+import { Status } from '../../types/Status';
 
 export const TodoList: React.FC = () => {
+  const todos = useAppSelector(state => state.todos);
+  const { status, query } = useAppSelector(state => state.filter);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useDispatch();
+  const filteredTodos = todos
+    .filter(todo => {
+      switch (status) {
+        case Status.Completed:
+          return todo.completed;
+        case Status.Active:
+          return !todo.completed;
+        default:
+          return todo;
+      }
+    })
+    .filter(todo =>
+      todo.title.toLowerCase().includes(query.toLocaleLowerCase()),
```

Comment:
There is a typo in 'toLocaleLowerCase'. It should be 'toLowerCase'. This could cause unexpected behavior when filtering todos by query.